google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/mutaterows/MutateRowsAttemptCallable.java

```diff
@@ -288,6 +292,26 @@ private void handleAttemptSuccess(List<MutateRowsResponse> responses) {
       }
     }
 
+    // Handle missing mutations
+    for (int i = 0; i < seenIndices.length; i++) {
+      if (seenIndices[i]) {
```

Comment:
nit: I find this a bit more readable
if (!seenIndices[i]) {
  // failure logic
}

Reply:
I generally disagree...I find the extra nesting to be harder to read, but its 2 against 1.

Reply:
Ok, so changing it led to me introducing a bug. Which makes me think the original form is easier to read. So I'm going to leave it as is